src/components/Products/SidebarFilterView/SidebarFilter.jsx

```diff
@@ -67,8 +67,13 @@ const SidebarFilter = () => {
                     <div className="category-filter">
                         <div className="list-title px-4 my-2 bold">Cateogry</div>
                         <div className="checklist">
-                            {Object.entries(categories).map(item =>{ 
-                                const [categoryName, isChecked] = item;
+                            {categories.map(item => {
+                                let categoryName='';
+                                let isChecked = false; 
+                                for(const c in item.selectedCategory){
```

Comment:
Didn't understand the purpose of this logic, Also this logic part can be removed from jsx part to make the code more cleaner.